healthcare/api-client/dicom/dicomweb_test.py

```diff
@@ -34,9 +34,12 @@
 RESOURCES = os.path.join(os.path.dirname(__file__), 'resources')
 dcm_file_name = 'dicom_00000001_000.dcm'
 dcm_file = os.path.join(RESOURCES, dcm_file_name)
-# The study_uid is not assigned by the server and is part of the
-# metadata of dcm_file
+# The study_uid, series_uid, and instance_uid are not assigned by the
+# server and are part of the metadata of dcm_file
 study_uid = '1.3.6.1.4.1.11129.5.5.111396399361969898205364400549799252857604'
+series_uid = '1.3.6.1.4.1.11129.5.5.195628213694300498946760767481291263511724'
+instance_uid = '1.3.6.1.4.1.11129.5.5.' + \
```

Comment:
Use format instead of string continuation:

instance_uid = '{}.{}'.format(
    '1.3.6.1.4.1.11129.5.5',
    '153751009835107614666834563294684339746480')

Reply:
Done. Please merge when you can.